components/shared_code/src/shared_data_model/meta/entity.py

```diff
@@ -27,6 +27,7 @@ class EntityAttributeType(StrEnum):
     DATETIME = "datetime"
     FLOAT = "float"
     INTEGER = "integer"
+    INTEGER_PERCENTAGE = "integer_percentage"
```

Comment:
Should we do any type checking on these attributes, i.e. `0 <= integer_percentage <= 100` ?

Reply:
Nope. The percentage is provided by the collector, which should be a trusted source.